exec/api/src/main/java/org/jboss/windup/engine/predicates/RuleProviderWithDependenciesPredicate.java

```diff
@@ -58,7 +58,7 @@ public boolean accept(WindupRuleProvider type)
                 List<String> executeAfterIDs = ruleProvider.getExecuteAfterIDs();
                 if ((executeAfter.contains(type.getClass())) || executeAfterIDs.contains(type.getID()))
                 {
-                    LOG.info("Accepting provider: " + type.getID());
+                    LOG.fine("Accepting provider: " + type.getID());
```

Comment:
I actually liked it as "info", at least for the "Skipping" one. Sometimes it can skip a provider that needs to run, and this is mostly used in tests. But it's no real problem either way, IMO.